spec/support/helpers/session_helpers.rb

```diff
@@ -1,7 +1,8 @@
 module Features
   module SessionHelpers
-    def sign_up_with(email, password, confirmation)
+    def sign_up_with(email, password, confirmation, username)
       visit new_user_registration_path
+      fill_in 'Username', with: username
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.